optaplanner-test/pom.xml

```diff
@@ -29,25 +29,6 @@
       <artifactId>optaplanner-core</artifactId>
     </dependency>
     <!-- External dependencies -->
-    <!-- Common utils -->
-    <dependency>
-      <groupId>org.apache.commons</groupId>
-      <artifactId>commons-lang3</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>com.google.guava</groupId>
-      <artifactId>guava</artifactId>
-    </dependency>
-    <!-- Logging -->
-    <dependency>
-      <groupId>org.slf4j</groupId>
-      <artifactId>slf4j-api</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>ch.qos.logback</groupId>
-      <artifactId>logback-classic</artifactId>
```

Comment:
Yes, I eventually agree with keeping this. Other modules have this dependency even though it's unused because the tests don't log anything. But in this case I think it's better to have the logging enabled in the test scope so when you add tests and need logging you don't have to figure out how to enable it.